### PR TITLE
Setting default file logging mode

### PR DESCRIPTION
**Pull request by fabiocav**

### CODE REVIEW

**Comment by fabiocav** [comment]
@brettsam merged this to get a test build, but will address concerns, if there are any

File: test/WebJobs.Script.Tests/Configuration/ScriptHostOptionsSetupTests.cs [hunk 1]
```diff
@@ -256,6 +256,31 @@ public void ConfigureLanguageWorkers_Default_IfNotDynamic_NoMaxMessageLength()
...
+            Assert.Equal(FileLoggingMode.DebugOnly, options.FileLoggingMode);
+        }
+
+        [Theory]
+        [InlineData("never", FileLoggingMode.Never)]
+        [InlineData("always", FileLoggingMode.Always)]
+        [InlineData("debugOnly", FileLoggingMode.DebugOnly)]
+        public void ConfigureW_WithConfiguration_AppliesFileLoggingMode(string setting, FileLoggingMode expectedMode)
+        {
+            var settings = new Dictionary<string, string>
```
[the remark this answers is not in the document]
Comment (brettsam) [reply test/WebJobs.Script.Tests/Configuration/ScriptHostOptionsSetupTests.cs]: I want to move this again -- we have "logger" (with only this setting) and "logging" (with all of our filtering and logger settings), which is confusing. .NET Core typically puts all logging settings under "logging", so we should just move this there and consolidate.

**Comment by fabiocav** [comment]
Sounds reasonable to me. It is a breaking change in config, but it won’t be the only one, so this is the time to get this right.

Again /cc @tdykstra

**Comment by brettsam** [comment]
I'll submit a PR for this. I also need to bind to the "aggregator", which is currently not working. I'll do both of those at the same time.

**Pull request merged**